ws/src/frida_language_processing/scripts/guest_analyzer.py

```diff
@@ -93,28 +98,31 @@ def guest_info_requested(self, request: GuestInfo) -> GuestInfoResponse:
             response.success = False
             return response
 
-        guest_name = text_completion.split(",")[0].strip().capitalize() # Extract the guest name
-        guest_drink = text_completion.split(",")[1].strip(' .').lower() # Extract the guest favorite drink
+        guest_name = text_completion.split(
+            ",")[0].strip().capitalize()  # Extract the guest name
+        guest_drink = text_completion.split(",")[1].strip(
+            ' .').lower()  # Extract the guest favorite drink
```

Comment:
check for len?

Reply:
Agree. My modifications in this doc only correspond to formatting (auto-formatter). Will not update as this service will be migrated to extract_data.py